pkg/usersettings/handlers.go

```diff
@@ -147,9 +146,8 @@ func (h *UserSettingsHandler) deleteUserSettings(ctx context.Context, client *ku
 
 func (h *UserSettingsHandler) createServiceAccountClient() (*kubernetes.Clientset, error) {
```

Comment:
What's the reason to keep this function compared to simply initializing the client during the `UserSettingsHandler` struct initialization?